test/blackbox-tests/test-cases/github1946.t/run.t

```diff
@@ -7,7 +7,7 @@ in the same dune file, but require different ppx specifications
   $ dune build @all --profile release
   $ dune ocaml merlin dump-config $PWD
   Usesppx1
-  ((STDLIB /OCAMLC_WHERE)
+  ((STDLIB /install_root/lib/ocaml)
```

Comment:
It seems the behaviour of BUILD_PATH_PREFIX_MAP is broken. The setting of the env var above should make this get substituted for OCAMLC_WHERE.

Reply:
Actually not. The setting of the map in the Dune supersedes the existing setting. For paths not covered by the Dune mappings, the previous settings would take effect.

One difference between the current Dune mappings and the previous settings is that before, only paths under the PWD were mapped. Now paths under the opam root are also mapped (to /install_root/...). That is why the prior mappings are not effective. Perhaps these tests should have these mappings removed since they are no longer needed?